recipes/bindsnet/LICENSE

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
The LICENSE is part of the github archive so there is no need to add it here.

Reply:
Good catch, thanks!